cpp/ql/lib/upgrades/dd32242a870867a532bb0b2a88a6a917a5b4c26f/usertypes.ql

```diff
@@ -0,0 +1,17 @@
+class UserType extends @usertype {
+  string toString() { none() }
+}
+
+bindingset[kind]
+int getKind(int kind) {
+  if kind in [10, 11, 12]
```

Comment:
Rows with 10, 11, 12 should not have been generated for a long time. However, before this PR there's no upgrade script that replaces these numbers with something within the used range of numbers. This means we could not remove any references to 10, 11, and 12. By adding this to this upgrade script we can now remove those references.